docker/release/windowsservercore/Dockerfile

```diff
@@ -1,48 +1,45 @@
 # escape=`
 FROM microsoft/windowsservercore
-MAINTAINER brycem@microsoft.com
-LABEL Readme.md="https://github.com/PowerShell/PowerShell/blob/master/docker/README.md"
-LABEL Description="This Dockerfile will install the latest release of PS."
+LABEL Readme.md="https://github.com/PowerShell/PowerShell/blob/master/docker/README.md",`
+      Description="This Dockerfile will install the latest release of PS."
 
-ARG POWERSHELL_MSI=https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-alpha.14/PowerShell_6.0.0.14-alpha.14-win10-x64.msi
-ARG POWERSHELL_SHA256=503F3AD52223699765895D3E9615FBD7988194693BCB725BE90C9EF0CD594447
+ARG POWERSHELL_MSI=https://github.com/PowerShell/PowerShell/releases/download/v6.0.0-alpha.15/PowerShell_6.0.0-alpha.15-win10-win2k16-x64.msi
```

Comment:
@brycem Please pick up the latest release MSI here
https://github.com/PowerShell/PowerShell/releases/latest

Then the experience would be similar to Linux. There we simply apt-get install / yum install from packages.microsoft.com and the image is automatlically populated with latest PS

This comment applies all locations where the MSI is downloaded